pre_commit/clientlib.py

```diff
@@ -156,6 +156,30 @@ def check(self, dct: Dict[str, Any]) -> None:
             )
 
 
+class OptionalSensibleSlashesAtHook(cfgv.OptionalNoDefault):
+    def check(self, dct: Dict[str, Any]) -> None:
+        super().check(dct)
+
+        if r'[\/]' in dct.get(self.key, ''):
+            logger.warning(
+                f'Pre-commit normalizes the slashes in {self.key!r} field in '
+                f"hook {dct.get('id')!r} to forward slashes, so you can use "
+                f'/ instead of [\\/]',
+            )
```

Comment:
I would just add these to the existing SensibleRegex checks

also this should check for both `r'[\/]'` and `r'[/\]'` since those are both valid spellings of "either slash"

Reply:
I've merged the check into the existing functions. However, `r[/\]` is not a valid regex, `\]` is an escape sequence to enable things like `[...\]]`.

Reply:
`[/\\]` is though, with the same issue

Reply:
Ok, I'll add it. Although it seems unlikely to me, that someone would use '[/\\]' instead of just '[\/]'.

Reply:
I've kept it in three ifs. If you feel looping through the regexes in for loop would be more readable, let me know.